### ISSUE TITLE
UI glitch issue resolved?

### ISSUE THREAD

**Issue by raptor12bee** [goatfungus/NMSSaveEditor#904]
Has anyone solved this issue yet? Every time I open it up and select a file to edit, my geforce starts up and then the screen just glitches out. It resets if I go into smaller screen mode but the moment I click on something or let it sit for a little bit, it starts glitching out again. 
![image](https://github.com/goatfungus/NMSSaveEditor/assets/148188796/718eec1e-b6d3-4c81-b063-4a6c24ae765d)

**Comment by runlykhel** [comment]
Nope, and not likely to be, windows 11/microsoft not suporting java which pretty much shot this editor it the foot for any of us with updated/newest systems. I've since moved over to nomnom, almost the same with no problems.

**Comment by cjg67** [comment]
Stop stating false information rnlykhel.  You have absolutely no idea what you are doing or talking about.
If you bothered to read what I posted, it HAS BEEN resolved.  If you are unable to follow instructions, that is your problem as the provided solution has worked flawlessly for me ever since discovering it.

**Comment by nmsain03** [comment]
> Stop stating false information rnlykhel. You have absolutely no idea what you are doing or talking about. If you bothered to read what I posted, it HAS BEEN resolved. If you are unable to follow instructions, that is your problem as the provided solution has worked flawlessly for me ever since discovering it.

hello can u help me with a other issue?

**Comment by MGJerry** [comment]
Agreed with @cjg67 , been using this editor fine on my Win 11 laptop even with 7 different Java versions installed, 0 issue. Shouldn't talk like you know a lot when you have 0 knowledge imo.

**Comment by raptor12bee** [comment]
> Stop stating false information rnlykhel. You have absolutely no idea what you are doing or talking about. If you bothered to read what I posted, it HAS BEEN resolved. If you are unable to follow instructions, that is your problem as the provided solution has worked flawlessly for me ever since discovering it.

Would you mind sharing this fix with me?

**Comment by Captsalt** [comment]
PLEASE post the url or the "fix" for the issue. Thanks!

**Comment by kjboggs** [comment]
The bat file worked perfectly until recently and then the interface got stupid again
